src/course.yaml

```diff
@@ -14,15 +14,19 @@ topics:
   - key: introduction
     title: Introduction
     details: >+
-      Smart contract auditing is the process of reviewing and testing the source
-      code of a smart contract to identify any potential vulnerabilities or
-      security issues. This can include checking for errors in the code,
-      evaluating the contract's overall design and architecture, and testing the
-      contract's functionality to ensure that it performs as intended. The goal
-      of a smart contract audit is to ensure that the contract is secure and
-      reliable, and that it will function as intended when deployed on a
-      blockchain network.
-
+      DeSci is a movement creating public infrastructure for scientific knowledge through the Web3 stack. 
```

Comment:
The chapter we are covering is the introduction chapter. So please update the `detail`